src/Panel/Routes/DropdownRoutes.php

```diff
@@ -17,34 +15,17 @@ class DropdownRoutes extends Routes
 	protected static string $prefix = 'dropdowns';
 	protected static string $type = 'dropdown';
 
-	public function params(Closure|array $params): array
-	{
-		// support direct handler
-		if ($params instanceof Closure) {
-			$params = [
-				'action' => $params
-			];
-		}
-
-		// support old options handler
-		if (isset($params['options']) === true) {
-			$params['action'] = $params['options'];
-		}
-
-		return $params;
-	}
-
 	public function toArray(): array
 	{
 		$routes = [];
 
 		foreach ($this->routes as $name => $params) {
-			$params  = $this->params($params);
+			$params  = $this->params($params, 'options');
 			$pattern = $this->pattern($params['pattern'] ?? $name);
 
 			$routes[] = $this->route(
 				pattern: $pattern,
-				action:  $params['action'],
+				action:  $params['action'] ?? fn () => 'The dropdown action handler is missing',
```

Comment:
One could also think whether here we should use `$params['load']` then. But then we have to add some legacy mapping from `action` to `load` in ``Routes::params()`.

Reply:
@bastianallgeier Implemented it.